Apps/Admin/Client/Pages/SupportPage.razor.cs

```diff
@@ -207,7 +207,7 @@ public SupportUserRow(ExtendedSupportUser model)
 
             public string PersonalHealthNumber { get; }
 
-            public DateTime? LastLoginDateTime { get; }
+            public DateTime LastLoginDateTime { get; }
```

Comment:
I think this will break the dependent query as it doesn't return a login date for the delegates